roles/neutron-prerequisites/tasks/ubuntu.yml

```diff
@@ -6,9 +6,18 @@
     - ifenslave
     - vlan
 
+- name: Download gpg key for Platform9 neutron repo
+  get_url:
+    url: https://platform9-neutron.s3-us-west-1.amazonaws.com/ubuntu_latest/key.gpg
+    dest: /tmp/pf9-key.gpg
+- name: Install Platform9 gpg key
+  apt_key: 
+    file: /tmp/pf9-key.gpg
+    state: present 
+
 - name: Add Platform9 APT repository
   apt_repository:
-    repo: 'deb [trusted=yes] http://platform9-neutron.s3-website-us-west-1.amazonaws.com ubuntu/'
+    repo: 'deb [trusted=yes] http://platform9-neutron.s3-website-us-west-1.amazonaws.com/ubuntu_latest /'
```

Comment:
As per https://docs.platform9.com/openstack/preparing-ubuntu/preparing_ubuntu_server_for_neutron, you don't need trusted=yes here?

Reply:
Yes, but I had observed failures for Neen when this wasn't set and the apt install in hostagent.log 
 complained about the repository not being trusted..

Reply:
trusted=yes is not required since ubuntu_latest has signed files. can you please confirm for Neen you had the same path for apt_repository? Also, can you please confirm S3 paths from pf9-onboarding repo?